profiles/Alibaba (Dubai).cyberduckprofile

```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ Copyright (c) 2002-2017 iterate GmbH. All rights reserved.
+  ~ https://cyberduck.io/
+  ~
+  ~ This program is free software; you can redistribute it and/or modify
+  ~ it under the terms of the GNU General Public License as published by
+  ~ the Free Software Foundation, either version 3 of the License, or
+  ~ (at your option) any later version.
+  ~
+  ~ This program is distributed in the hope that it will be useful,
+  ~ but WITHOUT ANY WARRANTY; without even the implied warranty of
+  ~ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  ~ GNU General Public License for more details.
+  -->
+
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+    <dict>
+        <key>Protocol</key>
+        <string>s3</string>
+        <key>Vendor</key>
+        <string>alibaba-oss-me-east-1</string>
+        <key>Scheme</key>
+        <string>https</string>
+        <key>Description</key>
+        <string>Alibaba (Dubai)</string>
+        <key>Default Hostname</key>
+        <string>oss-me-east-1.aliyuncs.com</string>
```

Comment:
Inconsistent across profiles.
Sometimes `region.oss.aliyuncs.com` sometimes `oss-region.aliyuncs.com`.
Consolidate single variant.

Reply:
> 



> Inconsistent across profiles. Sometimes `region.oss.aliyuncs.com` sometimes `oss-region.aliyuncs.com`. Consolidate single variant.

This is intentional. When available we use an [endpoint that supports IPv6 to access OSS
](https://www.alibabacloud.com/help/en/doc-detail/31834.htm#section-iao-0i8-eel)

Reply:
😠 Why can't those services make everything consistent.